src/storage/txn/latch.rs

```diff
@@ -149,9 +194,16 @@ impl Latches {
     where
         H: Hash,
     {
+        // TODO: avoid Hasher construct
         let mut s = DefaultHasher::new();
         key.hash(&mut s);
-        (s.finish() as usize) & (self.size - 1)
```

Comment:
Need to update comments which mention "slot ID". Now a lock stores key hashes only instead of slot ID.